### ISSUE TITLE
How to parse xml?

### ISSUE THREAD

**Issue by vishalkrv** [rc0x03/node-osmosis#27]
I am easily able to parse HTML however facing issues with XML. My sample xml document looks like 
```xml
   <urlset>
         <url>
           <loc>http://www.abc.com/1</loc>
           <image:image>
              <image:loc>http:/www.abc.com/image1.png</image:loc>
              <image:caption><![CDATA[Title 1]]></image:caption>
           </image:image>
         </url>
         <url>
           <loc>http://www.abc.com/2</loc>
           <image:image>
              <image:loc>http:/www.abc.com/image2.png</image:loc>
              <image:caption><![CDATA[Title 2]]></image:caption>
          </image:image>
        </url>
   <urlset>
```

And I was trying with this code:

```javascript

osmosis.get('http://www.abc.com/sitemap.xml')
		.set([{
			'url':'url > loc',
			'image':'url > image:loc'
		}])
		.data(function(array){
			console.log(array);
		});

```

**Comment by rc0x03** [comment]
This is due to a namespacing issue with libxmljs. This issue will be fixed in osmosis 0.0.7.

**Comment by rc0x03** [comment]
Fixed in 0.0.7

**Comment by vishalkrv** [comment]
Thanks
